src/run.rs

```diff
@@ -46,15 +47,17 @@ pub fn run_full(cd: Option<&Path>, name: &str, args: &[&str], env: &[(&str, &str
     }
 }
 
-pub fn run_capture(cd: Option<&Path>,
-                   name: &str,
-                   args: &[&str],
-                   env: &[(&str, &str)])
-                   -> Result<(Vec<String>, Vec<String>)> {
+pub fn run_capture<S>(cd: Option<&Path>,
+                      name: &str,
+                      args: &[S],
+                      env: &[(&str, &str)])
+                      -> Result<(Vec<String>, Vec<String>)>
+    where S: Borrow<str>
```

Comment:
I _think_ if you made this `S: AsRef<OsStr>` (so it matches the `Command::args` signature), you could remove the `iter().map()` below

https://doc.rust-lang.org/std/process/struct.Command.html#method.args